Finished working on user creation

andela-oadebayo opened the pull request:
Login is almost Done
-when admin logs in only thn will he see the create user link
-i should have this done by tommorow

andela-oadebayo committed:
Finished working on user creation
Login is almost Done

daybaryour commented:
aite Bro. Will review soonest.

On Tue, Jun 23, 2015 at 1:54 AM, Adebayo Oluwadamilola <
notifications@github.com> wrote:

> Login is almost Done
> -when admin logs in only thn will he see the create user link
> 
> ## -i should have this done by tommorow
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2
> Commit Summary
> - Finished working on user creation
> 
> File Changes
> - _A_ app/Helpers/Helper.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-0
>   (22)
> - _M_ app/config/app.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-1
>   (3)
> - _M_ app/controllers/HomeController.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-2
>   (56)
> - _M_ app/routes.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-3
>   (8)
> - _M_ app/start/global.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-4
>   (1)
> - _A_ app/views/Home/home.blade.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-5
>   (9)
> - _A_ app/views/Home/login.blade.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-6
>   (30)
> - _A_ app/views/Home/master.blade.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-7
>   (64)
> - _A_ app/views/Home/register.blade.php
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-8
>   (42)
> - _M_ composer.json
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-9
>   (8)
> - _A_ public/css/bootstrap.css
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-10
>   (6800)
> - _A_ public/css/style.css
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-11
>   (34)
> - _A_ public/fonts/glyphicons-halflings-regular.eot
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-12
>   (0)
> - _A_ public/fonts/glyphicons-halflings-regular.svg
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-13
>   (288)
> - _A_ public/fonts/glyphicons-halflings-regular.ttf
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-14
>   (0)
> - _A_ public/fonts/glyphicons-halflings-regular.woff
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-15
>   (0)
> - _A_ public/fonts/glyphicons-halflings-regular.woff2
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-16
>   (0)
> - _A_ public/js/bootstrap.js
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-17
>   (2363)
> - _A_ public/js/jquery.js
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-18
>   (10351)
> - _A_ public/js/npm.js
>   https://github.com/andela-oadebayo/Hospital_Inventory/pull/2/files#diff-19
>   (13)
> 
> Patch Links:
> - https://github.com/andela-oadebayo/Hospital_Inventory/pull/2.patch
> - https://github.com/andela-oadebayo/Hospital_Inventory/pull/2.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/andela-oadebayo/Hospital_Inventory/pull/2.

andela-oadebayo committed:
Admin can login and do crud on users

andela-oadebayo committed:
Different users can login to different places with the same login
Admin can do all
-Create users
-Do crud on users
-Create item
-Do crud on item

Mangers
-Can Do crud on item
